imagetool/src/main/java/com/oracle/weblogic/imagetool/logging/ConsoleFormatter.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package com.oracle.weblogic.imagetool.logging;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.logging.Formatter;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.LogRecord;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A color-capable formatter for the console that defines a fixed output format.
+ * Format is "[ LEVEL ] message", with optional throw-ables.
+ */
+public class ConsoleFormatter extends Formatter {
+    private static final String LINE_SEPARATOR = System.getProperty("line.separator");
+
+    static {
+        // if user is redirecting output, do not print color codes
+        if (System.console() == null) {
+            AnsiColor.disable();
+        } else {
+            // check logging properties to see if the user wants to disable color output
+            String flag = LogManager.getLogManager().getProperty("java.util.logging.ConsoleHandler.color");
+            if (flag != null && flag.equalsIgnoreCase("false")) {
+                AnsiColor.disable();
+            }
+        }
+    }
+
+    private AnsiColor getMessageLevelColor(Level level) {
+        if (level.intValue() > 800) {
+            return AnsiColor.BRIGHT_RED;
+        } else {
+            return AnsiColor.BRIGHT_BLUE;
+        }
+    }
+
+    private String replaceColorTokens(String text) {
+        //Pattern pattern = Pattern.compile("\\[([a-z]+)\\]");
```

Comment:
remove commented line.

Why not put this up in the class variables so it is only done once?

Reply:
done